src/telemetry.ts

```diff
@@ -24,18 +25,16 @@ export type TelemetryEvent = z.infer<typeof TelemetryEventSchema>;
 export type TelemetryEventTypes = TelemetryEvent['event'];
 export type EventProperties = TelemetryEvent['properties'];
 
+export const POSTHOG_KEY = 'phc_E5n5uHnDo2eREJL1uqX1cIlbkoRby4yFWt3V94HqRRg';
```

Comment:
do we definitely want to hardcode this or can we drop it in from an env variable at build time?